ext/web/01_dom_exception.js

```diff
@@ -160,6 +148,26 @@ class DOMException {
   }
 }
 
+ObjectDefineProperty(DOMException.prototype, "stack", {
+  get() {
+    return this[_error].stack;
+  },
+  set(value) {
```

Comment:
Added a setter since it was previously defined as `writable: true`, but I'm not really sure why it was defined this way.